docs/customization/spiders.rst

```diff
@@ -56,14 +54,15 @@ lower price:
     from scrapy_poet import DummyResponse
     from scrapy_spider_metadata import Args
     from zyte_common_items import Product
-    from zyte_spider_templates import EcommerceSpider, EcommerceSpiderParams
+    from zyte_spider_templates import EcommerceSpider
+    from zyte_spider_templates.spiders.ecommerce import EcommerceSpiderParams
 
 
     class MyParams(EcommerceSpiderParams):
         min_price: str = "0.00"
 
 
-    class MySpider(Args[MyParams], EcommerceSpider):
+    class MySpider(EcommerceSpider, Args[MyParams]):
```

Comment:
The old order caused "TypeError: Cannot create a consistent method resolution order (MRO) for bases Args, EcommerceSpider"